src/main/java/Arsdorint/Arsdorint.java

```diff
@@ -0,0 +1,336 @@
+import Arsdorint.task.Event;
+import Arsdorint.task.Task;
+import Arsdorint.task.Todo;
+
+public class Arsdorint {
+    private static final int MAX_NUM_OF_TASKS = 100;
+    private static final String EXIT_MESSAGE = " Bye. Hope to see you again soon!\n";
+    private static final String HELLO_MESSAGE =
+            " Hello! I'm Arsdorint, a member of Arsdorint Team.\n" +
+            " Please Type The Command As Follow:\n";
+    private static final String COMMAND_LIST_MESSAGE =
+            "> Type \"list\" to list all the tasks. \n" +
+            "> Type \"mark\" follow by a number x to mark x tasks in the list. \n" +
+            "> Type \"unmark\" follow by a number y to unmark y tasks in the list. \n" +
+            "> Type \"todo\" follow by a string x to add a work that need to be done. \n" +
+            "> Type \"deadline /x y\" with x is the type of work, y is the time or date of the deadline. \n" +
+            "> Type \"event x /y\" with x is the event, y is the time or date of that event. \n" +
+            "> Type \"bye\" to exit. \n";
+    private static final String QUESTION = " What can I do for you?";
+
+    private static final String STORAGE_DIRECTORY = "./storage";
+    private static final String STORAGE_FILE_NAME = "./storage/arsdorintTask.txt";
+    private static final String MESSAGE_NEW_FILE = "File created";
+    private static final String MESSAGE_OVERWRITE_FILE = "File overwritten";
+    private static final String MESSAGE_LOAD_FILE = "File loaded";
+    private static final String MESSAGE_NO_FILE = "There is no existing file";
+    private static final String MESSAGE_WRONG_FILE = "The storage files entry is invalid. Please save to overwrite";
+    private static final String MESSAGE_DIVIDER =
+            "____________________________________________________________";
+    private static final String MESSAGE_DIVIDER_LIST =
+            "____________________________LIST____________________________";
+    private static final String MESSAGE_DELETE = "Noted. I've removed this task:";
+
+    private static final String ERROR_MESSAGE_BYE = " ";
+    private static final String ERROR_MESSAGE_LIST = " ";
+    private static final String ERROR_MESSAGE_MARK =
+            "=( OOPS!!! The description of a mark cannot be empty.\n" + "Syntax for mark\n\t" +
+                    ">>> mark <item index number> \n" + "Note: item index must exist in the current list";
+    private static final String ERROR_MESSAGE_UNMARK =
+            "=( OOPS!!! The description of an unmark cannot be empty.\n" + "Syntax for unmark\n\t" +
+                    ">>> unmark <item index number> \n" + "Note: item index must exist in the current list";
+    private static final String ERROR_MESSAGE_TODO =
+            "=( OOPS!!! The description of a todo cannot be empty.\n" + "Syntax for todo\n\t>>> todo <task>";
+    private static final String ERROR_MESSAGE_DEADLINE =
+            "=( OOPS!!! The description of a deadline cannot be empty.\n" +
```

Comment:
You can consider relocating all your messages to a separate java file in order to improve code readability of `Arsdorint.java`.